islands/InfoCardSlider.tsx

```diff
@@ -44,62 +44,54 @@ export default function InfoCardSlider({
         <InfoCard
           {...props}
           typeOfContent={enrichedTypeOfContent}
+          styles={{
+            children: {
+              padding: "0",
+              minHeight: "initial",
+              backgroundColor: textBackgroundColor || undefined,
+            },
+          }}
           classes={{
             container: `${
               props?.direction === "left"
                 ? "flex-col lg:flex-row"
                 : "flex-col-reverse lg:flex-row-reverse"
             } w-full justify-start items-center`,
             children:
-              `infocard-children-container h-full flex flex-col items-start justify-center gap-[16px]`,
+              "infoCardChildren h-full flex flex-col items-start justify-center",
             button: `flex w-full pt-[0px] text-[14px]`,
             childrenTextContent: `${
               props?.link?.text ? "h-auto" : "h-full"
-            } w-full`,
-          }}
-          styles={{
-            children: {
-              backgroundColor: textBackgroundColor || undefined,
-              color: textColor || undefined,
-            },
+            } w-full infoCardInner`,
```

Comment:
@renan-s-coelho Sugestão:
```suggestion
            } w-full info-card-inner`,
```